client/oauth.go

```diff
@@ -71,7 +73,14 @@ func (c *ClientCredentials) updateConfig(r OAuthResponse) {
 
 // Perform a POST request.
 func (c *ClientCredentials) post(uri string, data url.Values) (*[]byte, error) {
-	res, err := http.PostForm(uri, data)
+	client := &http.Client{
+		Transport: &http.Transport{
+			TLSClientConfig: &tls.Config{
+				InsecureSkipVerify: c.InsecureSSL,
+			},
+		},
+	}
```

Comment:
This looks like a continuation of [this commit](https://github.com/foundriesio/fioctl/pull/239/commits/f93f30446e9593b928410c69f813973f96551072).

I _believe_ there is some common part here that can be generalized in a nice way; so that we don't have to copy it over and over again.

Reply:
If unification creates a problem for you; we're fine as is. We only have 2 places of a code like this.